app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Api
+  module V1
+    class UsersController < ApiController
+      helper_method :user
+
+      def show; end
+
+      def profile
```

Comment:
Looks like that `profile` method is exactly the same than `show`. 

The REST way to get specific info about a resource is using `show` method.
https://edgeguides.rubyonrails.org/routing.html#crud-verbs-and-actions

So you can get rid of this method and just use `show`.